src/api/Methods.ts

```diff
@@ -30,12 +30,17 @@ const UnavailableError = new Error('FDC3 DesktopAgent not available at `window.f
 const TimeoutError = new Error('Timed out waiting for `fdc3Ready` event.');
 const UnexpectedError = new Error('`fdc3Ready` event fired, but `window.fdc3` not set to DesktopAgent.');
 
+/**
+ * @deprecated This function depends on window.fdc3 (which may not be set for web-based desktop agents) 
+ * and does not wait on the fdc3Ready event, so it may return errors on container-based desktop agents.
```

Comment:
```suggestion
 * @deprecated This function depends on window.fdc3 (which may not be set for web-based Desktop Agents) 
 * and does not wait on the fdc3Ready event, so it may return errors on container-based Desktop Agents.
```

Reply:
applied